tssc/step_implementers/push_artifacts/maven.py

```diff
@@ -38,8 +81,7 @@ def _validate_step_config(self, step_config):
         if 'url' not in step_config or not step_config['url']:
             raise ValueError('url must have none empty value in the step configuration')
 
-    @staticmethod
```

Comment:
huh... wonder why this was ever static...

Reply:
because @ppremru had implmented it that way because it turns out we had a validate step config, but that wasn't the runtime step config, so i moved her logic up to the step_implementer level and made it not static.